docs/source/noise/index.rst

```diff
@@ -114,51 +114,51 @@ Noise types for each column
     - Types of noise
     - Notes
   * - First name
-    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
+    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, 1040, SSA
     - Leave a field blank, use a fake name, nickname, make typos, make OCR errors, make phonetic errors
-    -
+    - In the 1040 form, this noise type functions the same for joint filer and dependent *n*
```

Comment:
I think "noise type" isn't very clear in this context, since this table is about columns each of which have _multiple_ noise types. I'm also not sure our users will get "dependent n".

Here's one idea:

```suggestion
    - In the 1040 form, the same noise types are applied to the corresponding columns for the joint filer and dependents
```